File: polocloud-node/src/main/java/dev/httpmarco/polocloud/node/commands/type/StringArrayArgument.java
package dev.httpmarco.polocloud.node.commands.type;

import dev.httpmarco.polocloud.node.commands.CommandArgument;

public final class StringArrayArgument extends CommandArgument<String> {

    public StringArrayArgument(String key) {
        super(key);
    }

    @Override
    public String buildResult(String input) {
        return input;
    }
}

File: agent/src/main/java/dev/httpmarco/polocloud/agent/runtime/local/terminal/commands/type/StringArrayArgument.kt
package dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.type

import dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.CommandArgument
import dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.CommandContext

class StringArrayArgument(key: String) : CommandArgument<String>(key) {

    override fun buildResult(input: String, context: CommandContext): String {
        return input
    }
}